### PR TITLE
Parallelize contract deployments

### PR DESCRIPTION
**Pull request by mslipper**

### CODE REVIEW

**Comment by tynes** [comment]
devnet failed with:

```
Error: cannot estimate gas; transaction may fail or may require manual gas limit [ See: https://links.ethers.org/v5-errors-UNPREDICTABLE_GAS_LIMIT ] (reason="execution reverted", method="estimateGas", transaction={"from":"0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266","to":"0x8A791620dd6260079BF849Dc5567aDC3F2FdC318","value":{"type":"BigNumber","hex":"0x0de0b6b3a7640000"},"data":"0x9a2ac6d5000000000000000000000000f39fd6e51aad88f6f4ce6ab8827279cfffb922660000000000000000000000000000000000000000000000000000000000030d4000000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000000","accessList":null}, error={"name":"ProviderError","code":-32000,"_isProviderError":true}, code=UNPREDICTABLE_GAS_LIMIT, version=providers/5.6.8)
    at Logger.makeError (/home/circleci/project/node_modules/@ethersproject/logger/src.ts/index.ts:261:28)
    at Logger.throwError (/home/circleci/project/node_modules/@ethersproject/logger/src.ts/index.ts:273:20)
    at checkError (/home/circleci/project/node_modules/@ethersproject/providers/src.ts/json-rpc-provider.ts:78:20)
    at EthersProviderWrapper.<anonymous> (/home/circleci/project/node_modules/@ethersproject/providers/src.ts/json-rpc-provider.ts:603:20)
    at step (/home/circleci/project/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:48:23)
    at Object.throw (/home/circleci/project/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:29:53)
    at rejected (/home/circleci/project/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:21:65)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  reason: 'execution reverted',
  code: 'UNPREDICTABLE_GAS_LIMIT',
  method: 'estimateGas',
  transaction: {
    from: '0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266',
    to: '0x8A791620dd6260079BF849Dc5567aDC3F2FdC318',
    value: BigNumber { _hex: '0x0de0b6b3a7640000', _isBigNumber: true },
    data: '0x9a2ac6d5000000000000000000000000f39fd6e51aad88f6f4ce6ab8827279cfffb922660000000000000000000000000000000000000000000000000000000000030d4000000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000000',
    accessList: null
  },
  error: ProviderError: execution reverted
      at HttpProvider.request (/home/circleci/project/node_modules/hardhat/src/internal/core/providers/http.ts:78:19)
      at LocalAccountsProvider.request (/home/circleci/project/node_modules/hardhat/src/internal/core/providers/accounts.ts:188:34)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
      at async EthersProviderWrapper.send (/home/circleci/project/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)
}
```

It is possible that the metering logic will make gas estimation for deposits require a buffer, if the metering logic requires more gas to be burnt than when originally doing the gas estimation

**Comment by tynes** [comment]
Re running the devnet, we have experienced a revert a few times locally and never got to the root cause

**Comment by mslipper** [comment]
It's happened twice now, so there's likely an underlying issue. Any ideas where I should look first?

**Comment by tynes** [comment]
The messenger in the standard bridge may not be set correctly: https://github.com/ethereum-optimism/optimism/blob/3aacbee53c7ebb05e95e09e356cd76050ee38679/packages/contracts-bedrock/contracts/L1/L1StandardBridge.sol#L94

**Comment by mslipper** [comment]
Strange, all the checks still pass.

File: packages/sdk/src/utils/chain-constants.ts [hunk 1]
```diff
@@ -130,16 +130,16 @@ export const CONTRACT_ADDRESSES: {
   },
   [L2ChainID.OPTIMISM_BEDROCK_LOCAL_DEVNET]: {
     l1: {
-      AddressManager: '0x5FbDB2315678afecb367f032d93F642f64180aa3' as const,
```
Comment (tynes) [inline packages/sdk/src/utils/chain-constants.ts:133]: Perhaps we want to use `getL1ContractsByNetworkName('devnetL1')`

**Review by tynes — APPROVED** [review]

**Comment by tynes** [comment]
@mslipper Needs a lint then its good to go

File: packages/sdk/src/utils/chain-constants.ts (hunk 1 above)
Comment (mslipper) [reply packages/sdk/src/utils/chain-constants.ts:133]: I didn't see that in use for the non-Bedrock devnet, so I was unsure if we should convert over to it. Can follow up separately if that's the new best practice.

**Pull request merged**